dali/operators/reader/coco_reader_op.cc

```diff
@@ -76,11 +78,11 @@ The value is represented as an absolute value.)code",
   .AddOptionalArg("ratio",
       R"code(If set to True, the returned bbox and masks coordinates are relative to the image size.)code",
       false)
-  .AddOptionalArg("file_list",
+  .AddOptionalArg<string>("file_list",
```

Comment:
You can remove file_list altogether from here. It is not used. There is an error thrown if you provide it

Reply:
I'll remove it then.